pywhat/identifier.py

```diff
@@ -12,24 +12,42 @@ def __init__(self):
 
     def identify(self, text: str, api=False) -> dict:
         identify_obj = {}
+        magic_numbers = {}
+        file_text = []
+
+        if not api and (os.path.isdir(text) == True or os.path.isfile(text) == True):

```

Comment:
You can use `if not api and os.path.exists(text):`. The code will be shorter.

Reply:
Good idea! Will change it :)